examples/fabric-sync/admin/DeviceManager.h

```diff
@@ -0,0 +1,99 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "PairingManager.h"
+
+#include <app-common/zap-generated/cluster-objects.h>
+#include <platform/CHIPDeviceLayer.h>
+
+class DeviceManager : public PairingDelegate
```

Comment:
I always thought that this class could benefit with documentation to let reader know this manages devices on the remote bridge and not on the local bridge.

Do you think it is worth adding now?

Reply:
In fabric-admin, DeviceManager manages both local bridge and remote bridge, in fabric-sync, it currently only manages the remote bridge for 1-1 fabric-sync, but we might also need to manage the local bridge later, lets keep it as it is now.